test/clt-tests/core/test-freeze-unfreeze-update.rec

```diff
@@ -26,19 +26,18 @@ mysql -P9315 -h0 -e "select * from t;"
 |    1 | a    |
 +------+------+
 ––– input –––
-rm -f /tmp/update.sql; for n in `seq 1 10000`; do echo "update t set s='b' where id=1;" >> /tmp/update.sql; done; while true; do mysql -P9315 -h0 < /tmp/update.sql | break; done &
+(rm -f /tmp/update.sql; for n in `seq 1 10000`; do echo "update t set s='b' where id=1;" >> /tmp/update.sql; done; for i in `seq 1 10`; do mysql -P9315 -h0 < /tmp/update.sql; done; 2>&1 &)
```

Comment:
This takes 5 sec for me:
```
➜  ~ time (rm -f /tmp/update.sql; for n in `seq 1 10000`; do echo "update t set s='b' where id=1;" >> /tmp/update.sql; done; for i in `seq 1 10`; do mysql -P9306 -h0 < /tmp/update.sql; done &; wait)
( rm -f /tmp/update.sql; for n in `seq 1 10000`; do; echo  >> /tmp/update.sql)  0.06s user 0.28s system 6% cpu 5.539 total
➜  ~
```

I'm not sure it's safe enough to rely on this in the following commands. I.e. as I understand it's important that the updates are running in background throughout the whole test, but it may not be the case.

Reply:
Fixed